enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -149,6 +149,10 @@ const (
 	// This deadline isn't used for the Exec() codepath because removal is handled
 	// in the background by the runner pool.
 	finalizationTimeout = 10 * time.Second
+
+	// Max recv message size for Exec() requests, which can have lots of stderr/
+	// stdout in the response.
+	grpcMaxRecvMsgSize = 50_000_000
```

Comment:
could you add the units in the const name?

Reply:
Done